src/platform/packages/shared/kbn-server-route-repository/src/register_routes.ts

```diff
@@ -192,7 +192,12 @@ export function registerRoutes<TDependencies extends Record<string, any>>({
             ...options,
             access,
           },
-          security,
+          security: security ?? {
```

Comment:
@elena-shostak FYI I'm making security required here in [this PR](https://github.com/elastic/kibana/pull/216196)